circle.yml

```diff
@@ -0,0 +1,29 @@
+---
+machine:
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+  node:
+    version: 8.2.0
```

Comment:
Why 8.2.0? Could be just 8 instead (8.x is the current LTS)

Reply:
@xcarpentier This should simply be `8`, that's why it failed

Reply:
No, that failed because it’s v1 and you should specify strict version, soon I will change it to v2!

Reply:
I'm using it on this way:

`version: '8.9'`